cabal-install/tests/UnitTests/Distribution/Solver/Modular/Solver.hs

```diff
@@ -253,10 +253,11 @@ tests = [
         , runTest $ mkTest dbBuildable2 "choose version that sets buildable to false" ["A"] (solverSuccess [("A", 1), ("B", 2)])
          ]
     , testGroup "Pkg-config dependencies" [
-          runTest $ mkTestPCDepends [] dbPC1 "noPkgs" ["A"] anySolverFailure
```

Comment:
I think it would be useful to also keep the empty pkg-config test case, using `Just []`.